### DISCUSSION TITLE
Anyone working on e2e encryption?

### DISCUSSION THREAD

**Discussion by ameneres** [supabase/supabase#2297] [Feature Requests]
The idea is to have some data only be human readable (decrypted) on the frontend.

**Comment by stefroussos** [comment]
I'm building an E2EE project on Supabase. The crypto libraries i'm using are [crypto-js](https://github.com/brix/crypto-js) for secret key cryptography and [openpgpjs](https://github.com/openpgpjs/openpgpjs) & [tweetnacl-js](https://github.com/dchest/tweetnacl-js) for public key cryptography.

**Reply by ameneres** [reply]
That's awesome. Would love to check your project if open source or some kind of architecture diagram. I'll look into those projects. I'm not super familiar with E2EE but it's something I'd like to have for a project.
